modules/nlu/src/backend/tools/seeded-lodash.ts

```diff
@@ -0,0 +1,20 @@
+import _ from 'lodash'
+import seedrandom from 'seedrandom'
+
+export const getSeededLodash = (randomSeed?: number | string) => {
+  let lo = _
+  if (!randomSeed) {
+    return lo
+  }
+
+  const seed = _.isString(randomSeed) ? parseInt(randomSeed) : randomSeed
```

Comment:
This might fail. In the case of a string that is not a Number you'll get a NaN which in JS is a number ... will this make your seed crash deeper in bindings ?

Reply:
No it's fine, I tested that scenario